packages/beacon-node/src/chain/stateCache/fifoBlockStateCache.ts

```diff
@@ -68,6 +68,21 @@ export class FIFOBlockStateCache implements BlockStateCache {
     }
   }
 
+  /**
+   * Get a seed state for state reload.
+   */
+  getSeedState(): CachedBeaconStateAllForks {
```

Comment:
Why is only the first value important and not searching through for some condition?  May be helpful to add a more detailed comment about how and why this function is used.

Reply:
this could be any states, will add a comment for it